test/jdk/java/nio/file/WatchService/LotsOfEvents.java

```diff
@@ -140,6 +145,12 @@ static void drainAndCheckOverflowEvents(Path dir,
                 List<WatchEvent<?>> events = key.pollEvents();
                 System.err.printf("Retrieved key with %d events after %d ns%n",
                     events.size(), timeAfterPoll - timeBeforePoll);
+                // count for each kind of event
+                Map<WatchEvent.Kind, Long> countPerEventType = events.stream()
+                        .collect(Collectors.groupingBy(WatchEvent::kind, Collectors.counting()));
+                System.err.println("After extra polling, found:");
```

Comment:
I assume the trace message at L151 is not needed. L146 will print the count and this will be followed by the count of each kind.

Reply:
Done. Removed the (previous) L151 log message that was newly introduced in this PR.